shell/components/ResourceTable.vue

```diff
@@ -184,24 +194,31 @@ export default {
   },
 
   data() {
-    const options = this.$store.getters[`type-map/optionsFor`](this.schema);
-    const listGroups = options?.listGroups || [];
-    const listGroupMapped = listGroups.reduce((acc, grp) => {
-      acc[grp.value] = grp;
-
-      return acc;
-    }, {});
-
     // Confirm which store we're in, if schema isn't available we're probably showing a list with different types
     const inStore = this.schema?.id ? this.$store.getters['currentStore'](this.schema.id) : undefined;
 
-    return {
-      listGroups, listGroupMapped, inStore
-    };
+    return { inStore };
   },
 
   computed: {
+    options() {
+      return this.$store.getters[`type-map/optionsFor`](this.schema, !!this.externalPaginationEnabled);
```

Comment:
Why !!this.externalPaginationEnabled, it looks like externalPaginationEnabled is already  boolean so a conversion shouldn't be needed.

Reply:
Good catch. At one point it wasn't, will update